modules/log_ingestion.s3.cft.yaml

```diff
@@ -0,0 +1,80 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Description: >
+  CloudFormation organizational template for provisioning
+  the necessary resources for the `cloud-logs`
+  component and the read-only role required to itneract with
+  the target organizational environment.
+
+Metadata:
+  AWS::CloudFormation::Interface:
+    ParameterGroups:
+      - Label:
+          default: "Sysdig Settings (Do not change)"
+        Parameters:
```

Comment:
I suppose not adding org support for s3 is intended?

Reply:
yea there is nothing to do differently for s3 org support.